identified.tex

```diff
@@ -16,18 +16,18 @@ \subsection{Identified}
   
 \subparagraph{The \sbolheading{displayId} property}
 \label{sec:displayId}
-The \sbol{displayId} property is an OPTIONAL identifier with a data type of \sbol{String}. This property is intended to be an intermediate between a URI and the \sbol{name} property that is machine-readable, but more human-readable than the full URI of an object.
+The \sbol{displayId} property is an OPTIONAL identifier with a data type of \sbol{String}. This property is intended to be an intermediate between a IRI and the \sbol{name} property that is machine-readable, but more human-readable than the full IRI of an object.
 
 If the \sbol{displayId} property is used, then its \sbol{String} value MUST be composed of only alphanumeric or underscore characters and MUST NOT begin with a digit.
```

Comment:
Can definitions be added for "alphanumeric", "underscore", and "digit"?

For example, [Python uses the following definition of "alphabetic"](https://docs.python.org/3/library/stdtypes.html#str.isalpha) (a component of "alphanumeric"):

> Alphabetic characters are those characters defined in the Unicode character database as “Letter”, i.e., those with general category property being one of “Lm”, “Lt”, “Lu”, “Ll”, or “Lo”. Note that this is different from the “Alphabetic” property defined in the Unicode Standard.

For "underscore" I think we probably mean Unicode U+005F. [Wikipedia's definition of underscore](https://en.wikipedia.org/wiki/Underscore) lists three possibilities though.

Reply:
That's correct, the one that's equivalent to ASCII underscore 0x5F